backends/arm/operator_support/pool_2d_support.py

```diff
@@ -26,8 +26,8 @@ def stride_check(strides: tuple[int, int]) -> bool:
 
 
 def dim_check(shape=torch.Size) -> bool:
-    check = shape[0] == 1
-    for dim in shape:
+    check = True
+    for dim in shape[1:]:
         check &= 1 <= dim <= 65536
     return check
```

Comment:
JFYI this function could be written more idiomatically as `return all([1 <= dim <= 65536 for dim in shape[1:]])`